modules/modules.nix

```diff
@@ -124,6 +124,7 @@ let
     (loadModule ./services/syncthing.nix { })
     (loadModule ./services/taffybar.nix { })
     (loadModule ./services/tahoe-lafs.nix { })
+    (loadModule ./services/taskwarrior-sync.nix { condition = hostPlatform.isLinux; })
```

Comment:
I'm not sure whether this module should be named `taskwarrior` instead for consistency with `programs/taskwarrior`. `taskwarrior-sync` seems more explicit to me though.